# HG changeset patch
# User Rik <rik@octave.org>
# Date 1473955154 25200
#      Thu Sep 15 08:59:14 2016 -0700
# Node ID ad815427f376d1efe883f64340f3c0f09f1757f3
# Parent  3564b6b6b8d1f78706db2da8708486919e51a3d2
Add 10*eps tolerance for linspace tests to pass on Windows (bug #49092).

* data.cc: Add 10*eps tolerance for linspace tests to pass on Windows.

diff --git a/libinterp/corefcn/data.cc b/libinterp/corefcn/data.cc
--- a/libinterp/corefcn/data.cc
+++ b/libinterp/corefcn/data.cc
@@ -5185,27 +5185,27 @@ if fewer than two values are requested.
 ## Test complex values
 %!test
 %! exp = [1+0i, 2-1.25i, 3-2.5i, 4-3.75i, 5-5i];
 %! obs = linspace (1, 5-5i, 5);
 %! assert (obs, exp);
 
 ## Test support for vectors in BASE and LIMIT
 %!assert (linspace ([1 2 3], [7 8 9]),
-%!        [linspace(1, 7); linspace(2, 8); linspace(3, 9)])
+%!        [linspace(1, 7); linspace(2, 8); linspace(3, 9)], 10*eps)
 %!assert (linspace ([1 2 3]', [7 8 9]'),
-%!        [linspace(1, 7); linspace(2, 8); linspace(3, 9)])
+%!        [linspace(1, 7); linspace(2, 8); linspace(3, 9)], 10*eps)
 %!assert (linspace ([1 2 3], 9),
-%!        [linspace(1, 9); linspace(2, 9); linspace(3, 9)])
+%!        [linspace(1, 9); linspace(2, 9); linspace(3, 9)], 10*eps)
 %!assert (linspace ([1 2 3]', 9),
-%!        [linspace(1, 9); linspace(2, 9); linspace(3, 9)])
+%!        [linspace(1, 9); linspace(2, 9); linspace(3, 9)], 10*eps)
 %!assert (linspace (1, [7 8 9]),
-%!        [linspace(1, 7); linspace(1, 8); linspace(1, 9)])
+%!        [linspace(1, 7); linspace(1, 8); linspace(1, 9)], 10*eps)
 %!assert (linspace (1, [7 8 9]'),
-%!        [linspace(1, 7); linspace(1, 8); linspace(1, 9)])
+%!        [linspace(1, 7); linspace(1, 8); linspace(1, 9)], 10*eps)
 
 ## Test class of output
 %!assert (class (linspace (1, 2)), "double")
 %!assert (class (linspace (single (1), 2)), "single")
 %!assert (class (linspace (1, single (2))), "single")
 
 ## Test obscure Matlab compatibility options
 %!assert (linspace (0, 1, []), 1)
